app/views/procedures/_form.html.erb

```diff
@@ -1,16 +1,22 @@
 <%= form_for @procedure do |p| %>
-<%= p.text_field :name, placeholder: "Name"%>
-<%= p.date_field :date, placeholder: "Date"%>
-<%= p.number_field :price, placeholder: "Price", min:0, step:0.01%>
-<div>
-   <select name='facility_id'>
-     <option>
-       <% @facilities.each do |f| %>
-         <option value="<%= f.id %>"><%= f.name %></option>
-       <% end %>
-     </option>
-   </select>
-   <%= link_to  "Add Facility",new_facility_path %>
+<div class="p-2"><%= p.text_field :name, placeholder: "Name"%></div>
+<div class="p-2"><%= p.date_field :date, placeholder: "Date"%></div>
+<div class="p-2"><%= p.text_field :description, placeholder: "Description"%></div>
+<div class="p-2"><%= p.number_field :price, placeholder: "Price", min:0, step:0.01%></div>
+<div class="p-2">
+  <select name='facility_id'>
+   <option>
+     <% @facilities.each do |f| %>
+        <option value="<%= f.id %>"><%= f.name %></option>
+     <% end %>
+   </option>
+  </select>
  </div>
- <%= p.submit "Submit"%>
- <%end%>
+<div class="p-2">
+  <%= p.submit 'Submit', class:'btn btn-outline-primary'%>
+</div>
```

Comment:
You should be able to add the `class='p-2'` to the list of classes already on the submit button to simplify the code.